src/main/kotlin/uk/gov/justice/digital/hmpps/notificationsalertsvsip/service/listeners/notifiers/PrisonVisitCanclledEventNotifier.kt

```diff
@@ -1,15 +1,22 @@
 package uk.gov.justice.digital.hmpps.notificationsalertsvsip.service.listeners.notifiers
 
 import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
 import org.springframework.stereotype.Component
+import uk.gov.justice.digital.hmpps.notificationsalertsvsip.service.NotificationService
 import uk.gov.justice.digital.hmpps.notificationsalertsvsip.service.listeners.events.DomainEvent
+import uk.gov.justice.digital.hmpps.notificationsalertsvsip.service.listeners.events.additionalinfo.VisitAdditionalInfo
 
 const val PRISON_VISIT_CANCELLED = "prison-visit.cancelled"
 
 @Component(value = PRISON_VISIT_CANCELLED)
 class PrisonVisitCancelledEventNotifier(
+  private val notificationService: NotificationService,
   private val objectMapper: ObjectMapper,
 ) : EventNotifier(objectMapper) {
   override fun processEvent(domainEvent: DomainEvent) {
+    val visitAdditionalInfo: VisitAdditionalInfo = objectMapper.readValue(domainEvent.additionalInformation)
+    LOG.debug("Enter processEvent visitAdditionalInfo Info:$visitAdditionalInfo")
```

Comment:
Minor comment - can be ignored as not the standard way.  Intellij shows a warning against this. It recommends using {} and passing a parameter.